### PR TITLE
Update DeploymentsModal.js

### PR DESCRIPTION
**Pull request by SuperOmario**
changed lowercase s to uppercase S

### CODE REVIEW

File: plugins/services/src/js/components/DeploymentsModal.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ class DeploymentsModal extends mixin(StoreMixin) {
         href={`#/services/overview/${encodeURIComponent(servicePath)}`}
         onClick={this.props.onClose}
       >
-        {`services${servicePath}`}
+        {`services${ServicePath}`}
```
Comment (nLight) [inline plugins/services/src/js/components/DeploymentsModal.js]: Hi @SuperOmario, thanks for the contribution! I bet it's the first `services` that has to be changed.

**Comment by SuperOmario** [comment]
@nLight I'm sorry Ive been tryna figure it out for a while but I don't know how. If you wouldnt mind could you explain it to me? GitHub is confusing...

**Comment by nLight** [comment]
Sorry @SuperOmario the issue has been resolved https://github.com/dcos/dcos-ui/pull/3344

**Comment by Strum355** [comment]
Out of curiosity, why didnt a maintainer merge this with a squash commit? Theres a dropdown menu for merging in Github PR pages where one can choose to squash commit

**Comment by nLight** [comment]
@Strum355 we do rebase and merge

**Pull request closed without merging**